internal/server/config_test.go

```diff
@@ -300,15 +298,18 @@ func TestLoadConfigEmpty(t *testing.T) {
 	err := s.loadConfig(Config{})
 	assert.NilError(t, err)
 
+	org, err := data.GetOrganization(s.DB(), data.ByName(models.DefaultOrganizationName))
+	assert.NilError(t, err)
```

Comment:
Minor, but related to #3025, I'm trying to avoid looking up the default org in other places.

You can get the default org without looking it here with this:

```go
org := s.db.DefaultOrg
```

Reply:
I meant to remove all these, thanks for catching this one